src/src/com/microsoft/aad/adal/SerializedTokenCacheItem.java

```diff
@@ -0,0 +1,40 @@
+package com.microsoft.aad.adal;
+
+import com.google.gson.annotations.SerializedName;
+
+final class SerializedTokenCacheItem {
+
+    private static final String TAG = SerializedTokenCacheItem.class.getSimpleName();
+
+    @SerializedName("authority")
+    private final String mAuthority;
+
+    @SerializedName("refreshToken")
+    private final String mRefreshToken;
+
+    @SerializedName("idToken")
+    private final String mIdToken;
```

Comment:
It's not JSON compliment today... Basically there is a custom serialization/deserialization performed on it.